addon/components/frost-select.js

```diff
@@ -69,7 +69,6 @@ export default Component.extend({
     autofocus: PropTypes.bool,
     disabled: PropTypes.bool,
     error: PropTypes.bool,
-    hook: PropTypes.string.isRequired,
```

Comment:
This isn't necessary because it is provided in the `frost-component` that this component extends.